pkg/log/log.go

```diff
@@ -128,7 +124,12 @@ func InitLogger(cfg *Config) error {
 	appLogger = Logger{logger.WithOptions(zap.AddStacktrace(zap.DPanicLevel))}
 	appLevel = props.Level
 	appProps = props
-
+	// init and set tidb slow query logger to stdout if log level is debug
+	if cfg.Level == "debug" {
+		slowQueryLogger := zap.NewExample()
+		slowQueryLogger = slowQueryLogger.With(zap.String("component", "slow query logger"))
+		logutil.SlowQueryLogger = slowQueryLogger
```

Comment:
what is SlowQueryLogger if not debug level?

Reply:
That reminds me, if it wasn't debug level, I would have used the direct global logger

Now I've changed the behaviour so that if it's not at debug level, we don't spit out the log

Reply:
Does tidb slow log useful for dm debug? Shall we always set it as Nop?